blocks/product-recommendations/product-recommendations.css

```diff
@@ -44,6 +39,21 @@ main .section>div.product-recommendations-wrapper {
     display: inline-block;
 }
 
+.product-recommendations .product-grid  .product-grid-item button {
+  border-radius: var(--shape-border-radius-3);
+  border:none;
+  padding: var(--spacing-xsmall) var(--spacing-medium);
+  cursor:pointer;
```

Comment:
Wondering why `npm run lint:css` does not complain about this. Could you please add the correct spacing here? :)

Reply:
Yes, oops.

I may just end up using the UI renderer to render a button from the SDK instead.